apps/geo/serializers.py

```diff
@@ -148,3 +152,28 @@ class Meta:
             'region_title', 'admin_level_level', 'admin_level_title',
             'parent'
         )
+
+
+class RegionGqSerializer(ProjectPropertySerializerMixin, UserResourceSerializer):
```

Comment:
We are also missing other fields which are available in existing endpoint
https://github.com/the-deep/server/blob/0c1dc335c25035ce40b6e55ad73fc4ba3deafa59/apps/geo/serializers.py#L35-L81

Reply:
@thenav56 dai Deep client requires creating a region first, followed by the admin level. However, I need to remove the dependency on creating the admin level.

Reply:
https://github.com/the-deep/server/blob/0c1dc335c25035ce40b6e55ad73fc4ba3deafa59/apps/geo/serializers.py#L40-L45

@thenav56 dai in the existing rest api admin_level is in readonly field shoud we add in mutation as well?